src/main/java/org/sagebionetworks/bridge/services/AccountService.java

```diff
@@ -266,7 +273,10 @@ public void updateAccount(Account account) {
         if (!RequestContext.get().isAdministrator()) {
             account.setNote(persistedAccount.getNote());
         }
-
+        if (IS_ONLY_DEVELOPER.check()) {
+            Set<String> newDataGroups = addToSet(account.getDataGroups(), TEST_USER_GROUP);
+            account.setDataGroups(newDataGroups);
+        }
```

Comment:
This is probably redundant and be removed because getAccount(...) was already called, or it should throw UnauthorizedException.